### PR TITLE
Prepare 0.15.3 release

### PR DESCRIPTION
**Pull request by ameknite**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+- Automate `asio-sys` build.
+- Fix aio-sys build failure. Update `bindgen` in `asyo-sys` to 0.64.
+- Fix crash on web/wasm when `atomics` flag is enabled.
+- Improve Examples: Migrate wasm example to `trunk`, Improve syth-thones example.
+- Improve CI: Update actions, USe Android 30 in CI, Remove `asmjs-unknown-emscripten` target.
+- Update `windows` dependency to v0.54
+- Update `jni` dependency to 0.21
+- Update `alsa` dependency to 0.9
+- Update `oboe` dependency to 0.6
+- Update `ndk` dependency to 0.8 and Disable `default-features`.
```
Comment (est31) [inline CHANGELOG.md]:
```suggestion
- Update `ndk` dependency to 0.8 and disable `default-features`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,24 @@
 # Unreleased
 
-- Update `windows` dependency to v0.48
+# Version 0.15.3 (2024-03-02)
+
+- Add `try_with_sample_rate` a non-panicking variant of `with_sample_rate`.
```
Comment (est31) [inline CHANGELOG.md]:
```suggestion
- Add `try_with_sample_rate`, a non-panicking variant of `with_sample_rate`.
```

**Comment by est31** [comment]
1. why is it a draft? Do you have things you want to do before?
2. could you squash the history to one commit?

**Comment by ameknite** [comment]
1. I was testing if it works correctly with bevy, to avoid publishing another version.
2. Yeah.

I think is ready.

**Review by est31 — APPROVED** [review]
Thanks for the great PR!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+# Version 0.15.3 (2024-03-03)
+
+- Add `try_with_sample_rate`, a non-panicking variant of `with_sample_rate`.
+- struct `platform::Stream` is now #[must_use].
+- enum `SupportedBufferSize` and struct `SupportedStreamConfigRange` are now `Copy`.
+- `platform::Device` is now `Clone`.
+- Remove `parking_lot` dependency in favor of the std library.
+- Remove one_cell in `asio-sys`. Added call to `ASIOOutputReady`.
+- Automate `asio-sys` build.
+- Fix aio-sys build failure. Update `bindgen` in `asyo-sys` to 0.64.
```
Comment (est31) [inline CHANGELOG.md]: the three items above don't belong into the changelog I think, because they are asio-sys changes. Which this PR also doesn't release.

Could you remove these changes and make a separate PR that bumps the release version in the asio-sys crate?

Comment (ameknite) [reply CHANGELOG.md]: yes
